packages/server/src/services/agent.service.ts

```
@@ -33,14 +33,32 @@ export class AgentService implements IAgentService {
+  async handleUserRequest(
+    agent: AgentSystem | any,
+    userRequest: MessageRequest
   ): Promise<AgentExecutionResponse> {
     this.logger.debug({
       message: 'Processing agent request',
       request: userRequest.user_request,
     });
 
     try {
-      const result = await agent.execute(userRequest);
+      let result: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)